src/api2/middlewares/change-signers.js

```diff
@@ -0,0 +1,71 @@
+import { TransactionBuilder } from '../../base/transaction_builder'
```

Comment:
it's not middleware, move it somewhere near manager

Reply:
Moved to `SignersManager` class